index.html

```diff
@@ -143,54 +143,82 @@
 			<section id="terminology">
 				<h3>Terminology</h3>
 
+				<p>Wherever appropriate, this document relies on the terminology defined by the note on “Publishing and Linking on the Web”[[publishing-linking]]. In particular, the terms that are very important for this document include <a class="externalDFN" href="https://www.w3.org/TR/publishing-linking/#dfn-user">user</a>, <a class="externalDFN" href="https://www.w3.org/TR/publishing-linking/#dfn-user-agent">user agent</a>, <a class="externalDFN" href="https://www.w3.org/TR/publishing-linking/#dfn-browser">browser</a>, or <a class="externalDFN" href="https://www.w3.org/TR/publishing-linking/#dfn-address">address</a>.</p>
+
 				<dl>
-					<dt><dfn>Default Reading Order</dfn></dt>
-					<dd><p>The default reading order is a specific progression through the <a>primary resources</a> defined in
-						the <a>manifest</a> by the creator of a <a>Web Publication</a>.</p>
-						<p>A user might follow alternative pathways through the content, but in the absence of such interaction
-							the default reading order defines the expected progression from one primary resource to the
-							next.</p></dd>
+					<dt><dfn>URL</dfn></dt>
+					<dd>
+						<p>
+							In this specification, the general term URL is used as defined and used in other W3C specification like HTML&nbsp;[[html51]], and is defined by the relevant WhatWG document&nbsp;[[url]].
```

Comment:
Thanks. Comments on comments:
* I have changed to "Standard from WhatWG", we shall see what the W3C publishing rules will say about this.
* The reference to URL is in a non-normative section, hence the reference itself cannot be normative (per pubrules...)

Otherwise done.